packages/block-library/src/gallery/gallery-image.js

```diff
@@ -213,6 +213,7 @@ class GalleryImage extends Component {
 					tabIndex="0"
 					aria-label={ ariaLabel }
 					ref={ this.bindContainer }
+					aria-describedby={ 'caption-' + id }
```

Comment:
Our [coding guidelines](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#strings) say to use template literals whenever possible.

```suggestion
					aria-describedby={ `caption-${ id }` }
```